src/components/MovieCard/MovieCard.tsx

```diff
@@ -1,7 +1,51 @@
 import React from 'react';
+import { Movie } from '../../types/Movie';
 
 import './MovieCard.scss';
 
-export const MovieCard: React.FC = () => (
-  <>Put the card here</>
-);
+type Props = {
+  film: Movie;
+};
+
+export const MovieCard: React.FC<Props> = (props) => {
+  const { film } = props;
+  const {
+    title,
+    description = '',
```

Comment:
why do we need to add this default value here?

Reply:
I understand that in this case it's unnecessary, because undefined value won't be shown on the page, but it's more understandable for me. And if in future someone want to set some string as default value, he will only add here the needed value